Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Question/SeedDatabase.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Promact.Trappist.DomainModel.ApplicationClass;
+using Promact.Trappist.DomainModel.Models.Question;
+using Promact.Trappist.Web.Data;
+using System;
+using System.Linq;
+
+namespace Promact.Trappist.DomainModel.Data
+{
+
+    /// <summary>
+    /// Polpulates pre-required table
+    /// </summary>
+    public class SeedDatabase
+    {
+        public static object ProgramingLanguage { get; private set; }
+
+        /// <summary>
+        /// Seed data into the table
+        /// </summary>
+        /// <param name="context">DbContext</param>
+        public static void Seed(TrappistDbContext context) {
+            context.Database.EnsureCreated();
+
+            if (context.CodingLanguage.Any())
+            {
+                return;
+            }
+
+            var Languages = new CodingLanguage[]
+            {
+                new CodingLanguage {Language = ProgramingLanguageEnum.java,  CreatedDateTime = DateTime.UtcNow, UpdateDateTime = DateTime.UtcNow },
```

Comment:
please remove createddatetime and updatedateime assignments. it has been handled in dbcontext.